extensions/objects/templates/fraction-editor.directive.html

```diff
@@ -1,5 +1,11 @@
-<input type="text" ng-model="$ctrl.localValue.label" class="form-control">
+<input type="text" ng-model="$ctrl.localValue.label" class="form-control" ui-validate="'$ctrl.isValid($value)'">
```

Comment:
`isValid` function name is to generic please make it specific add some context on what this function validates.

Reply:
Done!